### PR TITLE
Modal content are not left aligned in I.E

### PR DESCRIPTION
**Pull request by vijayendranj**
I.E. and Edge browser doesn't support text-algin:start. I fixed it by switching using text-align:left
#798

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 ## 1.0.0-rc.5
```
Comment (paulcpederson) [inline CHANGELOG.md]: I think this should be `1.0.0-rc.6`?

**Comment by paulcpederson** [comment]
nice @vijayendranj !

File: CHANGELOG.md (hunk 1 above)
Comment (vijayendranj) [inline CHANGELOG.md]: My bad Copy/paste...Fixed  it

**Comment by vijayendranj** [comment]
@paulcpederson  Thx!

**Pull request merged**